src/wwwroot/template-editor/adapterService.js

```diff
@@ -396,9 +396,10 @@
           id: id,
           type: 'campaign',
           name: response.data.name,
+          html: response.data.html,
           attributes: response.data.mseditor ? response.data.mseditor.attributes : null,
-          innerHTML: response.data.mseditor ? response.data.mseditor.innerHTML : null,
-          children: response.data.mseditor ? response.data.mseditor.children : null
+          settings: response.data.mseditor ? response.data.mseditor.settings : null,
+          children: response.data.mseditor ? JSON.parse(response.data.mseditor.children) : null
```

Comment:
It should not be necessary.

Reply:
I guess it depends how to children is returned from backend.

Reply:
Right, if it is necessary to parse a JSON, there is something wrong.